content/docs/hello-world.md

```diff
@@ -15,36 +15,36 @@ ReactDOM.render(
 );
 ```
 
-It displays a heading saying "Hello, world!" on the page.
+Nó hiển thị một tiêu đề với dòng chữ "Hello, world!" trên trang web.
 
 [](codepen://hello-world)
 
-Click the link above to open an online editor. Feel free to make some changes, and see how they affect the output. Most pages in this guide will have editable examples like this one.
+Nhấn vào link bên trên để mở một trình soạn thảo trực tuyến. Hãy tự nhiên để làm vài sự thay đổi và thấy nó ảnh hưởng đến kết quả như thế nào. Hầu hết các trang trong hướng dẫn này sẽ có những ví dụ mà bạn có thể thay đổi được giống như ví dụ này.
 
 
-## How to Read This Guide {#how-to-read-this-guide}
+## Làm sao để đọc hướng dẫn này {#how-to-read-this-guide}
 
-In this guide, we will examine the building blocks of React apps: elements and components. Once you master them, you can create complex apps from small reusable pieces.
+Trong hướng dẫn này, chúng ta sẽ xem xét các khối xây dựng của các ứng dụng React: elements và components. Một khi bạn đã nắm vững chúng, bạn có thể tạo ra những ứng dụng phức tạp từ những mảnh ghép nhỏ tái sử dụng.
```

Comment:
Chị thấy câu này hơi kì kì, @nampdn xem có thể giúp chị sửa câu này hay hơn được hem nhé?

Reply:
Chỗ `elements` và `components` khi dịch từ này ra tiếng Việt em nghĩ nên để dạng bare verb (không cần s).
Em nghĩ khúc: `you can create complex apps from small reusable pieces.` => `Bạn có thể  tạo ra những ứng dụng lớn hơn từ những component nhỏ có tính tái sử dụng.`

Reply:
Gotcha!! Thank you em.

Reply:
Đã update. Review lại dùm chị nghen!

Reply:
`các khối xây dựng` em thấy nó hơi trừu tượng khi ra tiếng Việt, có thể dịch thành `cách xây dựng` không chị hen?
Đồng thời tránh lặp từ `các` ở phía sau nữa.

Reply:
Theo chị nghĩ từ "building block" mà mình dịch ra cách xây dựng thì nó hơi xa quá.

Reply:
Em cũng nghĩ vậy, nhưng nếu dịch sát quá thì lại khó hiểu, ngay cả dev làm việc mỗi ngày với React cũng khó có thể hình dung chỗ đó mình đang muốn nói điều gì nè.

Reply:
Cách xây dựng 1 ứng dụng React nó còn nhiều thứ liên quan, nếu dịch ra là cách xây dựng react trong tutorial này thì cũng chưa hẳn đúng. Chị đã đóng nháy và để từ tiếng anh đi kèm trong ý rồi. Sẽ update khi cần thiết.

Reply:
Ok chị nè ^^!